pep-0001.txt

```diff
@@ -229,45 +255,71 @@ PEP Review & Resolution
+a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the BDFL's delegate (or "PEP czar") for that PEP. If their self-nomination
-is accepted by the other core developers and the BDFL, then they will have
-the authority to approve (or reject) that PEP. This process happens most
-frequently with PEPs where the BDFL has granted in principle approval for
-*something* to be done, but there are details that need to be worked out
-before the PEP can be accepted.
-
-If the final decision on a PEP is to be made by a delegate rather than
-directly by the BDFL, this will be recorded by including the
-"BDFL-Delegate" header in the PEP.
-
-PEP review and resolution may also occur on a list other than python-dev
-(for example, distutils-sig for packaging related PEPs that don't
-immediately affect the standard library). In this case, the "Discussions-To"
-heading in the PEP will identify the appropriate alternative list where
-discussion, review and pronouncement on the PEP will occur.
+the BDFL-Delegate for that PEP, and they will then have the authority to approve
+(or reject) that PEP.
+
+The designated decision maker for each PEP is recorded in the "BDFL-Delegate"
+header in the PEP.
+
+Such self-nominations are accepted by default, but may be explicitly declined by
+the Steering Council. Possible reasons for the Steering Council declining a
+self-nomination as BDFL-Delegate include, but are not limited to, perceptions of
+a potential conflict of interest (e.g. working for the same organisation as the
+PEP submitter), or simply considering another potential BDFL-Delegate to be
+more appropriate. If core developers (or other community members) have concerns
+regarding the suitability of a BDFL-Delegate for any given PEP, they may ask
+the Steering Council to review the delegation.
+
+If no volunteer steps forward, then the Steering Council will approach core
+developers (and potentially other Python community members) with relevant
+expertise, in an attempt to identify a candidate that is willing to serve as
+BDFL-Delegate for that PEP. If no suitable candidate can be found, then the
+PEP will be marked as Deferred until one is available.
+
+While BDFL-Delegates hold final authority over acceptance or rejection of a PEP,
+they are expected to take the advice and perspectives of other core developers
+into account, and are free to seek additional guidance from the Steering Council
+at any time. BDFL-Delegates may also choose to step down, or be asked to step
+down by the Council, in which case a new BDFL-Delegate will be appointed in the
+same manner as for a new PEP (including deferral of the PEP if no suitable
+replacement can be found).
+
+When the PEP process is being used for projects other than the CPython
+reference interpreter, PEP review and resolution may also occur on a list
```

Comment:
I would like to explicitly include "Python the language" here.  IOW from:

"other than the CPython reference interpreter"

to

"other than the Python language or CPython reference implementation"

Maybe that's obvious, but I think it would be useful to be clear that discussions about the language irrespective of the implementation, also belong on python-dev.

Reply:
I ended up going the other direction, and just saying that cases like this require explicit Steering Council approval, rather than attempting to enumerate the situations where the Steering Council probably *wouldn't* give approval.